content/en/platform/corda/4.8/enterprise/operations/deployment/deployment-kubernetes.md

```diff
@@ -565,15 +564,15 @@ The Docker images used for the Kubernetes deployment are listed below for refere
 
 {{< table >}}
 
-| Service           | Image Name                           | Tag |
-|-------------------|--------------------------------------|-----|
-| Identity Manager  | acrcenm.azurecr.io/nmap/nmap         | 1.5 |
-| Network Map       | acrcenm.azurecr.io/nmap/nmap         | 1.5 |
-| Signing           | acrcenm.azurecr.io/signer/signer     | 1.5 |
-| Zone              | acrcenm.azurecr.io/zone/zone         | 1.5 |
-| Auth              | acrcenm.azurecr.io/auth/auth         | 1.5 |
-| Gateway           | acrcenm.azurecr.io/gateway/gateway   | 1.5 |
-| PKI Tool          | acrcenm.azurecr.io/pkitool/pkitool   | 1.5 |
-| Notary            | acrcenm.azurecr.io/notary/notary     | 1.5 |
+| Service           | Image Name                           | Tag   |
+|-------------------|--------------------------------------|-------|
+| Identity Manager  | acrcenm.azurecr.io/nmap/nmap         | 1.5.x |
```

Comment:
@YashNabar, should these all be:
corda/enterprise-nmap
corda/enterprise-signer
corda/enterprise-zone
... etc?

Reply:
In fact, shouldn't Identity manager be idman?

Reply:
Here's the list of docker images published to DockerHub as part of the 1.5.3 release (the string after the ":" is the tag)
```
corda/enterprise-identitymanager:1.5.3-zulu-openjdk8u242
corda/enterprise-networkmap:1.5.3-zulu-openjdk8u242
corda/enterprise-pkitool:1.5.3-zulu-openjdk8u242
corda/enterprise-signer:1.5.3-zulu-openjdk8u242
corda/enterprise-auth:1.5.3-zulu-openjdk8u242
corda/enterprise-gateway:1.5.3-zulu-openjdk8u242
corda/enterprise-zone:1.5.3-zulu-openjdk8u242
```
I also noticed a typo in the "Service" column for "Signing Service".

Reply:
@michelemarsden Here's the list following the 1.5.4 release

```
corda/enterprise-identitymanager:1.5.4-zulu-openjdk8u242
corda/enterprise-networkmap:1.5.4-zulu-openjdk8u242
corda/enterprise-pkitool:1.5.4-zulu-openjdk8u242
corda/enterprise-signer:1.5.4-zulu-openjdk8u242
corda/enterprise-auth:1.5.4-zulu-openjdk8u242
corda/enterprise-gateway:1.5.4-zulu-openjdk8u242
corda/enterprise-zone:1.5.4-zulu-openjdk8u242
corda/enterprise-notary:4.5.9-zulu-openjdk8u242
```